python/lsst/ts/standardscripts/auxtel/latiss_take_sequence.py

```diff
@@ -0,0 +1,295 @@
+                   a specified exposure time.
+                anyOf:
+                  - type: array
+                    minItems: 1
+                    items:
+                      type: number
+                      minimum: 0
+                  - type: number
+                    minimum: 0
+                default: 2.
+
+              reason:
+                description: Optional reason for taking the data.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+
+              program:
+                description: >-
+                  Optional name of the program this dataset belongs to.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+
+              do_track_target:
+                description: Track target during sequence.
+                type: boolean
+                default: True
+
+              do_check_ataos_corrections:
+                description: Check that ATAOS corrections are enabled before taking sequence.
+                type: boolean
+                default: True
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+
+        # make a list of tuples from the filter, exptime and grating lists
+        _recurrences = (
+            len(config.exposure_time_sequence)
+            if isinstance(config.exposure_time_sequence, collections.abc.Iterable)
+            else 1
+        )
+
+        self.visit_configs = [
+            (f, e, g)
+            for f, e, g in zip(
+                format_as_list(config.filter_sequence, _recurrences),
+                format_as_list(config.exposure_time_sequence, _recurrences),
+                format_as_list(config.grating_sequence, _recurrences),
+            )
+        ]
+
+        self.reason = config.reason
+
+        if self.reason is None:
+            warnings.warn(
+                "Reason not specified! Consider specifying the 'reason' for "
+                "this observation. it helps reducing the data aftwards. Running "
+                "without specifying a 'reason' will stop working in the future!",
+                DeprecationWarning,
+            )
+
+        self.program = config.program
+
+        if self.program is None:
+            warnings.warn(
+                "Program not specified! Consider specifying the 'program' for "
+                "this observation, it helps reducing the data aftwards. Running "
+                "without specifying a 'program' will stop working in the future!",
+                DeprecationWarning,
+            )
+
+        self.do_track_target = config.do_track_target
```

Comment:
I thought we agreed that this could be removed.. this script is designed to take data while tracking. So, this should not be optional.